ambry-account/src/main/java/com/github/ambry/account/InMemoryUnknownAccountService.java

```diff
@@ -35,6 +38,20 @@ public Account getAccountById(short accountId) {
   }
 
   @Override
+  public boolean addAccountUpdateConsumer(Consumer<Collection<Account>> accountUpdateConsumer) {
+    checkOpen();
```

Comment:
This is to make same behavior as the `HelixAccountService`. This is the api-level thing.